src/server/req_runjob.c

```diff
@@ -1999,8 +1998,7 @@ convert_job_to_resv(job *pjob)
 	}
 	newreq->rq_type = PBS_BATCH_SubmitResv;
 
-	get_jobowner(pjob->ji_wattr[(int)JOB_ATR_job_owner].at_val.at_str, owner);
-	strncpy(newreq->rq_user, owner, PBS_MAXUSER);
+	get_jobowner(pjob->ji_wattr[(int)JOB_ATR_job_owner].at_val.at_str, newreq->rq_user);
```

Comment:
Don't you need the user for request authentication?

Reply:
instead of having the value in a local variable "owner", I am directly getting it into newreq->rq_user.